### DISCUSSION TITLE
Trial extension

### DISCUSSION THREAD

**Discussion by hussein-yassine** [community/community#29552] [Copilot Conversations]
Hello all,

I applied to the Github Copilot two months ago, and it's not until yesterday that I saw the your Github copilot registration is not complete.
I completed the registration and now I only have till August 22 and the trail will end, with me not having proper time to really test this tool, so I just want to know if this trial can be extended

**Comment by mazkdevf** [comment]
Hello Hussein Yassine,

I'm not **working** on GitHub so this can be wrong info, But i don't think it is possible.

But also its mentioned on GitHub Copilot website

![image](https://user-images.githubusercontent.com/79049205/184690312-1e4fe024-f955-48ab-bf24-434b2f4ba8c6.png)

**Reply by mazkdevf** [reply]
So i'm not sure, but 🤷🏻‍♂️

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
